src/angularWhitespaceRule.ts

```diff
@@ -65,9 +76,42 @@ class InterpolationWhitespaceVisitor extends BasicTemplateAstVisitor implements
   }
 }
 
+class SemicolonTempalteVisitor extends BasicTemplateAstVisitor implements ConfigurableVisitor {
+
+  visitDirectiveProperty(prop: ast.BoundDirectivePropertyAst, context: BasicTemplateAstVisitor): any {
+
+    if (prop.sourceSpan) {
+      const directive = (<any>prop.sourceSpan).toString();
+      let expr = directive.split("=")[1].trim();
+      expr = expr.substring(1,expr.length-1).trim();
+
+      // Note that will not be reliable for different interpolation symbols
+      let error = null;
+
+      if (SemicolonNoWhitespaceRe.test(expr)) {
+        error = 'Missing whitespace after semicolon; expecting \'; expr\'';
+        const internalStart = expr.search(SemicolonNoWhitespaceRe)+1;
```

Comment:
@rokerkony yes, it's best if we add the checks as part of CI.

A PR will be good.